RELEASE-NOTES.md

```diff
@@ -1,3 +1,84 @@
+# Riak CS 1.5.4 Release Notes
+
+## Bugs Fixed
+
+- Disable previous riak object after backpressure sleep is triggered
+  [riak_cs/#1041](https://github.com/basho/riak_cs/pull/1041). This
+  change prevents unnecessary siblings increase when backpressure is
+  triggered under high upload concurrency, and uploads are interleaved
+  during the backpressure sleep. This issue does not affect multipart
+  uploads.
+- Fix wrong path rewrite in S3 API by unnecessary unquote
+  [riak_cs/#1040](https://github.com/basho/riak_cs/pull/1040). Due to
+  the wrong handling of URL encoding/decoding, object keys including
```

Comment:
For 1.3.0 release notes, there is a phrase using quote. So in English sense both could be common. But removed 'quote' and all will be 'encode/decode'.